bundles/AdminBundle/Controller/Searchadmin/SearchController.php

```diff
@@ -484,10 +484,13 @@ public function quicksearchAction(Request $request, EventDispatcherInterface $ev
                     $data['iconCls'] .= ' pimcore_icon_' . $element->getType();
                 }
 
+                $validLanguages = \Pimcore\Tool::getValidLanguages();
```

Comment:
twig extension here might be a good idea as well:`pimcore_valid_languages`

Reply:
Don't think this could be of general use, ... I'd just leave it for now as it is.

Reply:
I think it could, I've had use case for it.